docs/DeveloperGuide.md

```diff
@@ -622,38 +630,192 @@ testers are expected to do more *exploratory* testing.
 
    1. Download the jar file and copy into an empty folder
 
-   1. Double-click the jar file
+   2. Double-click the jar file
```

Comment:
Just sharing, there is no need to change the index to 2 because the markdown language will automatically change it to ordered list